templates/restic_script_Linux.j2

```diff
@@ -24,7 +32,7 @@ export B2_ACCOUNT_KEY={{ restic_repos[item.repo].b2_account_key }}
 {% if item.src is defined %}
 BACKUP_SOURCE={{ item.src }}
 {% endif %}
-set -euxo pipefail
+set -uxo pipefail
```

Comment:
I had to change this, otherwise an error in the backup/prune process would have prevented logging of the result.